### DISCUSSION TITLE
deno fmt is checking hundreds of files when I only have 5 modules in my project

### DISCUSSION THREAD

**Discussion by akatechis** [denoland/deno#15280] [Q&A] [answered]
Is it checking remote modules that I import as well? When I run `deno fmt --check` I get a message saying it checked 419 files.

**Comment by bartlomieju** [comment]
CC @dsherret

**Comment by dsherret** [comment, marked as answer]
Nope. Do you have json or markdown files? Is there a hidden folder in your directory tree? What version of deno are you using?

**Reply by akatechis** [reply]
Yeah, I think this is it... I have a `.cov` directory for coverage files which are `json`. I didn't think to look at it, because it's in my .gitignore

**Reply by dsherret** [reply]
Ok cool. You can exclude it if you'd like via a deno.json file: https://deno.land/manual/getting_started/configuration_file#example
